README.md

```diff
@@ -8,305 +8,409 @@
+Before you begin, complete the [prerequisite steps](README.md#prerequisites) to log in to IBM Cloud and your cluster, and optionally set up a [service ID API key](README.md#using-a-service-id).
 
-```bash
-curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash
-```
+To install the latest release for OpenShift before install, run the following script:
 
-The above script stores an API key in a Kubernetes secret that can be accessed by the operator.
-Next, it sets default values used in provisioning IBM Cloud Services, like the resource group and region.
-You can override any default value in the `Service` custom resource.
+*   **Latest release**: 
+    
+    ```bash
+    curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash
+    ```
 
-If you prefer to create the secret and the defaults manually, consult the [IBM Cloud Operator documentation](docs/install.md).
+*   **Specific release**: Replace `-v 0.0.0` with the specific version that you want to install.
+    
+    ```bash
+    curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- -v 0.0.0 store-creds
+    ```
 
-To configure with a specific version of the operator, you can pass a semantic version:
+<!-- END SHOW operator hub -->
+[Back to top](#ibm-cloud-operator)
 
-```bash
-curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- -v 0.0.0 store-creds
-```
+### Installing the operator for Kubernetes clusters
 
-## Using the IBM Cloud Operator
+Before you begin, complete the [prerequisite steps](#prerequisites) to log in to IBM Cloud and your cluster, and optionally set up a [service ID API key](#using-a-service-id).
 
-You can create an instance of an IBM public cloud service using the following custom resource:
+*   **Latest release**: To install the latest stable release of the operator, run the following script.
+    
+    ```bash
+    curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- install
+    ```
 
-```yaml
-apiVersion: ibmcloud.ibm.com/v1
-kind: Service
-metadata:
-    name: myservice
-spec:
-    plan: <PLAN>
-    serviceClass: <SERVICE_CLASS>
-```    
+*   **Specific release**: Replace `-v 0.0.0` with the specific version that you want to install.
+    
+    ```bash
+    curl -sL https://raw.githubusercontent.com/IBM/cloud-operators/master/hack/configure-operator.sh | bash -s -- -v 0.0.0 install
+    ```
 
-to find the value for `<SERVICE_CLASS>`, you can list the names of all IBM public cloud 
-services with the command:
+[Back to top](#ibm-cloud-operator)
 
-```bash
-ibmcloud catalog service-marketplace
-```
+### Uninstalling the operator
+Before you begin, complete the [prerequisite steps](#prerequisites) to log in to IBM Cloud and your cluster.
```

Comment:
We should add a note here that this will also delete the CRDs and their associated custom resources. It can be pretty destructive if they have a bunch of provisioned services.